DevOps/Readme.md

```diff
@@ -53,6 +53,7 @@
   - [ Docker Tutorial - Kunal Khushwaha(Youtube)](https://www.youtube.com/watch?v=17Bl31rlnRM)
   - [Docker Tutorial - TechWorld with Nana(Youtube)](https://www.youtube.com/watch?v=3c-iBn73dDE)
   - [Learn Docker - freecodecamp(Youtube)](https://www.youtube.com/watch?v=9zUHg7xjIqQ)
+  - [A practical guide on Docker - Hitesh Choudhary(Youtube)](https://www.youtube.com/watch?v=rr9cI4u1_88&t=4594s)
```

Comment:
Add the link below the Official Docker documentation link.

Reply:
@TRohit20 Sure